### PR TITLE
Fix  #4062 to allow more than one proxy server running by start scripts with different config directories.

### LINKED ISSUE #4062: Sharding-Proxy should be able to start more than one instance from the start-script
## Bug Report
Download the proxy binary zip file and extract to a proxy directory.
After start one proxy with start.bat or start.sh, run it with another port and config path, but some  occurs. 
Sharding-Proxy should start more than one instance from the start-script.

```
Exception in thread "main" com.atomikos.icatch.SysException: Error in init: Log already in use? xa_tx in ./logs\
        at com.atomikos.icatch.provider.imp.AssemblerImp.createRepository(AssemblerImp.java:181)
        at com.atomikos.icatch.provider.imp.AssemblerImp.assembleTransactionService(AssemblerImp.java:156)
        at com.atomikos.icatch.config.Configuration.assembleSystemComponents(Configuration.java:485)
        at com.atomikos.icatch.config.Configuration.init(Configuration.java:448)
        at com.atomikos.icatch.config.UserTransactionServiceImp.initialize(UserTransactionServiceImp.java:105)
        at com.atomikos.icatch.config.UserTransactionServiceImp.init(UserTransactionServiceImp.java:219)
        at org.apache.shardingsphere.transaction.xa.atomikos.manager.AtomikosTransactionManager.init(AtomikosTransactionManager.java:43)
        at org.apache.shardingsphere.transaction.xa.XAShardingTransactionManager.init(XAShardingTransactionManager.java:54)
        at org.apache.shardingsphere.transaction.ShardingTransactionManagerEngine.init(ShardingTransactionManagerEngine.java:69)
        at org.apache.shardingsphere.shardingproxy.backend.communication.jdbc.datasource.JDBCBackendDataSource.<init>(JDBCBackendDataSource.java:68)
        at org.apache.shardingsphere.shardingproxy.backend.schema.LogicSchema.<init>(LogicSchema.java:57)
        at org.apache.shardingsphere.shardingproxy.backend.schema.impl.ShardingSchema.<init>(ShardingSchema.java:81)
        at org.apache.shardingsphere.shardingproxy.backend.schema.LogicSchemaFactory.newInstance(LogicSchemaFactory.java:56)
        at org.apache.shardingsphere.shardingproxy.backend.schema.LogicSchemas.initSchemas(LogicSchemas.java:105)
        at org.apache.shardingsphere.shardingproxy.backend.schema.LogicSchemas.init(LogicSchemas.java:94)
        at org.apache.shardingsphere.shardingproxy.Bootstrap.startWithRegistryCenter(Bootstrap.java:116)
        at org.apache.shardingsphere.shardingproxy.Bootstrap.main(Bootstrap.java:83)
Caused by: com.atomikos.recovery.LogException: Log already in use? xa_tx in ./logs\
        at com.atomikos.persistence.imp.LogFileLock.acquireLock(LogFileLock.java:59)
        at com.atomikos.recovery.imp.FileSystemRepository.init(FileSystemRepository.java:55)
        at com.atomikos.icatch.provider.imp.AssemblerImp.createCoordinatorLogEntryRepository(AssemblerImp.java:229)
        at com.atomikos.icatch.provider.imp.AssemblerImp.createRepository(AssemblerImp.java:179)
        ... 16 more
```

### Which version of ShardingSphere did you use?
4.0.0 or 5.0.0-RC1-SNAPSHOT in dev branch

### Which project did you use? Sharding-JDBC or Sharding-Proxy?
Sharding-Proxy

### Expected behavior
use 
> start.bat 3307 
> start.bat 3308 /conf1/

Then will be starting two proxy server successfully.

### Actual behavior
Errors occur.

### Reason analyze (If you can)
Config path param isn't passed to ShardingConfigurationLoader.load method and xa log files conflict 
 in jta.properties file.

### Example codes for reproduce this issue (such as a github link).

As a result, we should: 

Step 1. download Sharding-Proxy and extract to {Sharding-Proxy-dir}, and then configure server.yaml and other config files in /config dir, run it with command start.bat or start.sh in /bin dir.

Step 2. cp {Sharding-Proxy-dir}/conf to {Sharding-Proxy-dir}/conf1, and add a file named `jta.properties` to /conf1 with content: 
```
com.atomikos.icatch.log_base_dir = ./logs1
```

Step 3.  Run another proxy-server with port 3308:
```
$ ./start.sh  3308  /conf1/
```
or in Windows: 
```
$ ./start.bat  3308  /conf1/
```
then the second proxy-server is running now.

### PR DESCRIPTION
**Pull request by kimmking**
Fixes #4062

Changes proposed in this pull request:
- allow more than one proxy server running by start scripts with different config directories.
- rename jta.properties to transactions.properties and make users can use their own jta.properties overwrite the first one and avoid the conflict between log file configs.
- polish the two script start.sh/start.bat.

### CODE REVIEW

**Comment by cherrylzhao** [comment]
@kimmking  transactions.properties could not be loaded by atomikos

File: sharding-proxy/sharding-proxy-bootstrap/src/main/java/org/apache/shardingsphere/shardingproxy/Bootstrap.java [hunk 1]
```diff
@@ -74,9 +76,12 @@
      * @throws SQLException SQL exception
      */
     public static void main(final String[] args) throws IOException, SQLException {
-        ShardingConfiguration shardingConfig = new ShardingConfigurationLoader().load();
-        logRuleConfigurationMap(getRuleConfiguration(shardingConfig.getRuleConfigurationMap()).values());
         int port = getPort(args);
+        String path = getConfigPath(args);
+
```
Comment (terrymanu) [inline sharding-proxy/sharding-proxy-bootstrap/src/main/java/org/apache/shardingsphere/shardingproxy/Bootstrap.java]: We should remove useless blank line.

Comment (kimmking) [reply sharding-proxy/sharding-proxy-bootstrap/src/main/java/org/apache/shardingsphere/shardingproxy/Bootstrap.java]: > We should remove useless blank line.

done.

File: sharding-proxy/sharding-proxy-bootstrap/src/main/java/org/apache/shardingsphere/shardingproxy/Bootstrap.java [hunk 2]
```diff
@@ -91,6 +93,18 @@ private static int getPort(final String[] args) {
...
     }
+
+    private static String getConfigPath(final String[] args) {
+        if (args.length < 2) {
+            return DEFAULT_CONFIG_PATH;
+        }
+        return paddingWithSlash(args[1]);
+    }
+
+    private static String paddingWithSlash(final String str) {
```
Comment (terrymanu) [inline sharding-proxy/sharding-proxy-bootstrap/src/main/java/org/apache/shardingsphere/shardingproxy/Bootstrap.java]: name of `str` maybe `arg`

**Pull request merged**